src/lib/y2network/virtualization_config.rb

```diff
@@ -72,14 +70,38 @@ def create
         bridge_builder.save
 
         # Move routes from the port member to the bridge (bsc#903889)
-        Yast::LanItems.move_routes(builder.name, bridge_builder.name)
+        move_routes(builder.name, bridge_builder.name)
       end
 
       true
     end
 
   private
 
+    # Adds a new interface with the given name
+    def add_device_to_routing(name)
+      return if @config.nil?
```

Comment:
it is an attr_reader, you can use 

```suggestion
      return unless config
```
```suggestion
      return if @config.nil?
```